Sources/Extraction/Filters/LocalHistogram.c

```diff
@@ -18,20 +18,23 @@ Int16Array3D LocalHistogram_Analyze(const BlockMap *blocks, const UInt8Array2D *
     return histogram;
 }
 
-Int16Array3D LocalHistogram_SmoothAroundCorners(const BlockMap *blocks, const Int16Array3D *input)
+Int16Array3D LocalHistogram_SmoothAroundCorners(const Int16Array3D *input)
 {
-    Int16Array3D output;
+    Int16Array3D output = Int16Array3D_Construct(input->sizeX + 1, input->sizeY + 1, input->sizeZ);
```

Comment:
I would say we don't add +1 here as clients may not expect this to be bigger than the input image size